src/extension.ts

```diff
@@ -112,7 +122,11 @@ class TestplaneExtension {
             }
 
             runProfile.runHandler = async (request, token): Promise<void> => {
-                await runner.runTests(request, token);
+                try {
+                    await runner.runTests(request, token);
+                } catch (err) {
+                    showTestplaneError("There was an error during run Testplane tests", err as Error);
```

Comment:
Here I need show error if something goes wrong. `Try catch` outside this method (`runHandler`) can't catch error from here.